plugins/modules/system/dconf.py

```diff
@@ -64,6 +64,7 @@
     default: present
     choices:
       - read
+      - list
```

Comment:
Adding a `list` value to `states` is not allowed (some legacy code does it, and these are always bad examples). Create an `_info` module instead (`dconf_info`), or `_facts` if appropriate.

Reply:
Thank you for your response. The state name does not matter. I changed list and fixed other bot errors
```
ansible -m dconf -a 'key=/org/gnome/ state=list_sub_dirs' localhost
[WARNING]: No inventory was parsed, only implicit localhost is available
localhost | SUCCESS => {
    "changed": false,
    "values": [
        "control-center/",
        "deja-dup/",
        "desktop/",
        "eog/",
        "evolution-data-server/",
        "nautilus/",
        "settings-daemon/",
        "shell/",
        "system/",
        "terminal/"
    ]
}
```

Reply:
The name does indeed not matter. A list functionality does not belong into the module, no matter how it is called.

Please create an `_info` module. This PR will not get merged, no matter how you call that state.

Reply:
+1 to @felixfontein 's statement
@wrcomb please close the pr and create a new one for _info module instead.
(Don't forget about unit and/or integration tests for it, all new modules must have them)